inst/include/common/data_object.hpp

```diff
@@ -51,6 +51,7 @@ struct DataObject : public fims_model_object::FIMSObject<Type> {
   size_t jmax;            /**< 2nd dimension of data object>*/
   size_t kmax;            /**< 3rd dimension of data object>*/
   size_t lmax;            /**< 4th dimension of data object>*/
+  Type na_value = -999;    /**< specifying the NA value >*/
```

Comment:
Noting that the NA commit snuck into this PR!

Reply:
Suggestion [non-blocking]: Maybe we can move your comments from the pull request #540 to here and keep changes in the prop_female branch? Before merging the code into the main branch, we can squash commits in this branch so that the pull request includes two distinct changes: one for handling missing data with -999, and the other for setting up the proportion female vector.

Reply:
Yes! I made a change to the vignette to address @k-doering-NOAA 's comments on the isNA change in this branch, and I will delete the other PR.